include/onnxruntime/core/common/common.h

```diff
@@ -283,4 +283,6 @@ inline std::wstring ToWideString(const std::wstring& s) { return s; }
 inline std::string ToWideString(const std::string& s) { return s; }
 #endif
 
+std::vector<std::string> SplitStr(const std::string s, const char splitor);
```

Comment:
>td::string s [](http://example.com/codeflow?start=41&length=12)

I think this should either be a non-const value or const reference

Reply:
Yes, done, thanks!